### ISSUE TITLE
Specifications

### ISSUE THREAD

**Issue by camipacifici** [camipacifici/fitlegac#1]
_(by Adam)_

- What is age? Is it the mean or median? is it the mean/median time at which all the stars formed (weighted integral under the SFH), or mean/median time at which the remaining stellar mass formed?

- r-band light weighted quantities needs to be measured in the rest-frame. Shall we use the actual SDSS r filter or a median flux in a certain range around 6250A?

- Stellar mass includes dark remnants. (If people can...if not please specify in your README)

- What format should the SFH take? I would advocate a list of ages with a set of draws from the posterior at each age. We could also agree on a common age grid to make things easier later.

- What format should the posterior spectra take? I would advocate a set of posterior flux predictions at each of the wavelength points in the DR3 spectra.

**Comment by siriobelli** [comment]
Good questions. I suggest that every "scalar" quantity (i.e. one number per galaxy: light-weighted age, mass, A_V, etc) should be given as 16, 50, 84 percentiles, following Adam's first commit. So in practice this means median and not mean values.

**Comment by siriobelli** [comment]
Never mind, I only now realized that I confused two completely different things: for scalar quantities we should give the 16, 50, 84 percentiles of the posterior distribution; but this does not affect the definition of the age.

If we want to weight the age by mass and light then it makes more sense to use the mean, since a weighted median doesn't have a straightforward definition or interpretation. Then we would give the 16, 50, 84 percentiles of the posterior distribution for the weighted-mean age (by mass and by light).

Although maybe it would also be interesting to compare the median age? Or something like 16, 50, 84 percentiles of the SFH?

**Comment by jrleja** [comment]
I think comparing just mass-weighted + light-weighted mean ages make sense.
I suspect that ultimately the most interesting & interpretable comparisons
will not be the one-point statistics, but instead looking at the full
SFR(t).

**Comment by ACCarnall** [comment]
I agree with Sirio and Joel's comments, the only further important thing to specify is what we mean by "mass" when we do mass-weighted quantities. In the results I've posted so far mass-weighted quantities are weighted by the formed mass, i.e. the age is calculated from the age-weighted integral under the SFH.

**Comment by siriobelli** [comment]
To be consistent with the definition of light-weighted age, I think we should consider only stars that are still present at the observed time? But then we can always define the actual stellar mass of the galaxy to include dark remnants, if we want. Do we want to?

On a related note, what do you guys mean exactly by SFH? If I do the integral of my SFH, I do not get the formed mass but the mass present now. In other words, my SFH is not SFR(t).

**Comment by ACCarnall** [comment]
I think your second comment is possibly the source of the confusion, for me, SFH = SFR(t). However I think I remember you saying you use a constant return fraction as a function of age? If this is true then your mass-weighted ages shouldn't be affected.

I don't think there's an issue in being consistent with light-weighted ages, the light-weighted age is the same whether or not you include dark remnants and material returned to the ISM.

It was also my understanding that the common definition of stellar mass included dark remnants, but not material returned to the ISM?

**Comment by siriobelli** [comment]
Yes, in my case SFR(t) = constant * SFH(t) because I use a constant return fraction (to Joel's great disappointment). However I usually work with quiescent galaxies, for which this approximation is OK. Since we have lots of star-forming galaxies in our sample now, I think this approximation is not very good anymore.

Since I use the Bruzual & Charlot library, my stellar masses also include dark remnants (and not gas returned to the ISM) (or at least that's what I think!), however I'm not sure this is a universal choice. What do other people think?

**Comment by siriobelli** [comment]
Now that I think about it, my SFH(t) actually *is* proportional to SFR(t), because I use the BC03 library, in which the spectra are normalized by the stellar mass formed at t=0 and not the stellar mass that survived. I never had to think about this because of my stupid instantaneous approximation. So now it seems that we agree on a few definitions:

1) SFH(t) = SFR(t). We should probably give the SFH(t) directly in units of Msun/yr so to avoid further confusion.

2) Galaxy stellar masses include dark remnants but not mass returned to the ISM. This is not directly related to the integral of SFH(t)

You are right to point out that the light-weighted age is not affected by dark remnants, so the most natural definition would be to weigh the age using the mass definition in 2). However this is different from taking the integral of the SFH(t), which includes the mass returned to the ISM. So we have two choices:

3a) The mass-weighted age is calculated using the mass of stars+remnants as weight (this is not straightforward to calculate, I think?)

3b) The mass-weighted age is calculated using the mass of stars+remnants+ejected gas as weight (i.e., by taking the SFH(t) as weight).

What are people's opinions on the matter?

**Comment by ACCarnall** [comment]
Good summary, I'm in favour of 3b as the easiest to explain/calculate

**Comment by camipacifici** [comment]
Thank you for the summary @siriobelli ! I was about to write the same.
I am also in favor of 3b for the mass-weighted MEAN age.

About the definition of SFH, lets just call it either SFH or SFR(t) from now on. I find SFH(t) very confusing. SFR(t) in M_sun/yr is perfect.

The "observed mass" includes the dark remnants, but not the mass returned to the ISM.
The "total mass" includes everything and is equivalent to the integral of SFR(t).
Normally we compare the observed mass.
